### PR TITLE
[MRG] feat(Cookie Policy Page): First implementation of the cookie policy page

### PR DESCRIPTION
**Pull request by joaobarcia**
Implements new cookie policy page.

https://app.asana.com/0/995964219886332/1184493948725137/f

### CODE REVIEW

File: site/content/english/cookie-policy-v2020-08-19.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
+---
+title: Cookie Policy
+slug: cookie-policy
+aliases:
+  - cookies/archive/20200819
+  - cookies-v2020-08-19
```
Comment (betatim) [inline site/content/english/cookie-policy-v2020-08-19.md:6]: Is there a reason to add this alias? For the privacy policy we have it but only for old versions in order to maintain a legacy URL. For legal documents that were never available at this kind of URL we should only have them at URLs following this pattern `/cookies/archive/20200819`.

Comment (Kirastina) [reply site/content/english/cookie-policy-v2020-08-19.md:6]: @joaobarcia can you take care of this? :)

Comment (joaobarcia) [reply site/content/english/cookie-policy-v2020-08-19.md:6]: @betatim , so your recommendation is to remove line 6, is that correct?

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested this and it looks good from my point of view.

**Pull request merged**